src/openforms/authentication/tasks.py

```diff
@@ -20,5 +22,6 @@ def update_saml_metadata() -> None:
     `digid_eherkenning.management.commands` module of django-digid-eherkenning library.
     Updates the stored metadata file and prepopulates the db fields.
     """
-    call_command("update_stored_metadata", "digid")
-    call_command("update_stored_metadata", "eherkenning")
+    stdout = StringIO()
```

Comment:
I'm using this to suppress the output in tests - when running the full test suite locally I don't want do say anything else other than `.`, `E`, `S` and `F`.

Otherwise we indeed do not store the result in the celery result store (as that requires you to clean it up at some point), but it's really just for the test output.